server/ai_mediaserver.go

```diff
@@ -48,6 +48,31 @@ const (
 	Complete   ImageToVideoStatus = "complete"
 )
 
+type ObjectDetectionResponseAsync struct {
```

Comment:
Are these being used?

Reply:
Yes, the struct type is being used [here](https://github.com/livepeer/go-livepeer/blob/a4f06d9f6b03afe23672834bc2352020106580bb/server/ai_mediaserver.go#L611)

Reply:
I think we should not add another async result endpoint.  I believe we will add a universal endpoint to check for async results at some point down the road rather than have async for some pipelines and not others.  @rickstaa what are your thoughts?

Reply:
I agree with you that generalisation is better to be done in the case async requests' result endpoints for all the pipelines equally. But right now for this PR should I keep the async functioning? @ad-astra-video What would you suggest?!
cc @rickstaa

Reply:
I would prefer remove the async part on this PR and we put on the road map a more general way to make all requests async if preferred by the user.

Reply:
Done in the latest commit :+1: